4141 remove f90 references from both docstrings and docs (#4283)
ukaea/PROCESS: documentation/source/development/add-vars.md

  **At all times the [`PROCESS` style guide](../development/standards.md) must be used.**

-----------------

## Add an input

To add a `PROCESS` input, please follow below:

1. Choose the most relevant module `XX` and add the variable in the `XX_variables` defined in `XX_variables.py`.
 
2. Add a description of the input variable below the declaration, using the formatting described in the standards section specifying the units.

> 3. Assign a sensible initial value as the default type.

Reviewer: ```suggestion
3. Assign a sensible default initial value, and a type.
```

  

`nsweep` comment example:
```python
  nsweep: int = None
  """Switch denoting quantity to scan:<UL>
  <LI> 1  aspect
  <LI> 2  pflux_div_heat_load_max_mw
  <LI> 3  p_plant_electric_net_required_mw
  <LI> 4  hfact
```

`nsweep_dict` example in `scans.py`
```python
nsweep_dict = {
        1: "aspect",
        2: "pflux_div_heat_load_max_mw",
        3: "p_plant_electric_net_mw",
        4: "hfact",
        ...
}
```

`ScanVariables` case example:

```python
  class ScanVariables(Enum):

>     @classmethod

Reviewer: I'm not sure if the `_missing_` method needs to be in this docs page?
